namseent.code-workspace

```diff
@@ -12,6 +12,9 @@
     {
       "path": "namui-animation-editor"
     },
+    {
+      "path": "namui-prebuilt"
+    },
```

Comment:
Prebuilt already added in [here](https://github.com/NamseEnt/namseent/pull/253/files?diff=split&w=0#diff-d91c10e5d39ce1e9af1b18e874c5850c28087235a7eee3d3123a8f25e190f6acR30-R32)